apps/web-giddh/src/app/services/sales.service.ts

```diff
@@ -113,4 +115,47 @@ export class SalesService {
             return data;
         }), catchError((e) => this.errorHandler.HandleCatch(e)));
     }
+
+    /**
+     * API call to get all advance receipt vouchers using invoice date
+     *
+     * @param {AdvanceReceiptRequest} model Request model
+     * @returns {Observable<BaseResponse<any, AdvanceReceiptRequest>>} API response
+     * @memberof SalesService
+     */
+    public getAllAdvanceReceiptVoucher(model: AdvanceReceiptRequest): Observable<BaseResponse<any, AdvanceReceiptRequest>> {
+        this.user = this._generalService.user;
+        this.companyUniqueName = this._generalService.companyUniqueName;
```

Comment:
Don't use companyUniqueName from general service. It will come wrong if we will switch company.

Reply:
Done, Have change in header as header will call every company change